changeSet.py

```diff
@@ -5,10 +5,10 @@
 import pmEnums
 
 # class Change:
-# 	def __init__(self, lineNum, changeType, data):
-# 		self.lineNum = lineNum
-# 		self.changeType = changeType
-# 		self.data = data	
+#	def __init__(self, lineNum, changeType, data):
```

Comment:
Your class declaration is still commented out so there will likely be a runtime error here when the module is imported.